.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  env: {
+    browser: true
+  },
+  extends: ['digitalbazaar', 'digitalbazaar/jsdoc'],
+  globals: {
```

Comment:
we should get most/all of these globals by setting `browser: true` as you've done here.  I'm merging this and I'll eliminate as many globals as possible.